casper/src/test/scala/coop/rchain/casper/engine/BlockApproverProtocolTest.scala

```diff
@@ -43,7 +43,7 @@ class BlockApproverProtocolTest extends FlatSpec with Matchers {
   }
 
   it should "log a warning for invalid ApprovedBlockCandidates" in effectTest {
-    val n = 8
+    val n = 3
```

Comment:
I assume this number was changed due to its "implicit" dependency on number of validators generated in GenesisParameters so maybe make it explicitly depend on it. In other words, change it to something like `genesisParams.genesis.bonds.size - 1`